_posts/2020-12-10-David_Microsoft.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: post
+title: Upcoming Why R Webinar - R at Microsoft
+published: false
+date: 2020-12-08 11:00:00
+author: Laís Carvalho
+categories: [webinar]
+tags: [2020]
+output:
+  html_document:
+    mathjax:  default
+    fig_caption:  true
+    toc: true
+    section_numbering: true
+    keep_md: true
+---
+
+<img src="/foundation/images/fulls/webinars/david_smith.jpg" class="fit image">
+
+This Thursday, December 10th at 7 pm UTC, as part of the Why R? Webinar series, we will be delighted to host [David Smith](https://www.linkedin.com/in/dmsmith/), Cloud Developer Advocate at [Microsoft](https://azure.microsoft.com/en-us/). He will be speaking about the history of R at Microsoft and surveying the company products that now integrate with our beloved R programming language.  
+David will answer all your questions at the end of the session. Join us!
+
+
+## Webinar
+
+<iframe width="850" height="480" src="https://https://youtu.be/ELUXXUf-NfI" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
@laisbsc in the url you have https://https:// twice. Also the url should be in a different format for embedding 
Pay attention to the embeded `embeded` "https://www.youtube.com/embed/ELUXXUf-NfI" part. Other than that this is ready to go. I'll update this part thank you.